src/Juvix/LLVM/Codegen/Constants.hs

```diff
@@ -0,0 +1,21 @@
+-- | Module for predefined constants
+module Juvix.LLVM.Codegen.Constants where
+
+import Juvix.Library hiding (Type)
+import qualified LLVM.AST.Type as Type
+
+portLength ∷ Num p ⇒ p
+portLength = 32
+
+-- | An abbreviation for 'IntegerType' 8
+i4 ∷ Type.Type
+i4 = Type.IntegerType 4
+
+-- TODO ∷ change this based on the machine, C++ pargma?
```

Comment:
You can use pragmas, but I don't know if we want to tie it to the machine Juvix is compiled on.